test/lit.cfg

```diff
@@ -833,3 +853,6 @@ config.substitutions.append(('%target-object-format', config.target_object_forma
 
 config.substitutions.append(('%target-resilience-test-wmo', config.target_resilience_test_wmo))
 config.substitutions.append(('%target-resilience-test', config.target_resilience_test))
+
+config.substitutions.append(('%target-llvm-profdata', os.path.join(llvm_tools_dir, 'llvm-profdata')))
+config.substitutions.append(('%target-llvm-cov', os.path.join(llvm_tools_dir, 'llvm-cov')))
```

Comment:
Okay, well then you *do* need to use the substitution. Setting things on `config` doesn't do anything on its own.